app/authentication/auth.py

```diff
@@ -50,6 +49,7 @@ def requires_auth():
         raise AuthError("Invalid token: iss not provided", 403)
 
     if client == current_app.config.get('ADMIN_CLIENT_USER_NAME'):
+        g.service_id = current_app.config.get('ADMIN_CLIENT_USER_NAME')
```

Comment:
does it make sense for us to try and grab [a] service of the user who is currently logged in? is there any way we can make that useful?